src/components/config/KeyboardShortcutsPanel.tsx

```diff
@@ -0,0 +1,117 @@
+import { Accordion, AccordionDetails, AccordionSummary } from "@mui/material";
+import Paper from "@mui/material/Paper";
+import Table from "@mui/material/Table";
+import TableBody from "@mui/material/TableBody";
+import TableCell from "@mui/material/TableCell";
+import TableContainer from "@mui/material/TableContainer/TableContainer";
+import TableRow from "@mui/material/TableRow/TableRow";
+import { type } from "@tauri-apps/api/os";
+import { Component } from "react";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+
+type Props = object;
+
+type State = object;
+
+let isMac = false;
+// We don't want to block during the imports stage, especially
+// if type() somehow fails.
+type().then(type=>{
+    isMac = (type == "Darwin");
+})
+// These default to the Mac ⌘ key because it's easier to replace with Ctrl than vice versa
+let shortcuts = {
```

Comment:
all of these should be flipped. I see people looking for what is the generate key to find the shortcut. not, oh i forgot what command G does?

tldr should be descriptions on left and the shortcut on right (flipped).

Reply:
Done